.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: A short description of the new feature
+labels: enhancement
+assignees: sinukaarel
```

Comment:
Same as with issue template - I would leave assignees out to not give the wrong signal that you are working on it.

Reply:
Removed assignee https://github.com/sendsmaily/smaily-opencart-module/pull/80/commits/df54c1e23ef01dafd971b5b6e948f1dca2b3c9f3